add requirements on eth rpc endpoint (#33)
dydxprotocol/v4-documentation: pages/validators/starting_the_network.md

For your node to successfully ingest bridge transactions from the relevant blockchain, you should use your own private RPC endpoint and override the default by adding flag `--bridge-daemon-eth-rpc-endpoint <YOUR_PRIVATE_RPC_ENDPOINT>` to the command you run when starting the node.

> The RPC endpoint you choose should satisfy below requirements:

Reviewer: ```suggestion
💡IMPORTANT💡:The RPC endpoint you choose *MUST* satisfy the following requirements
```
